src/pds_doi_service/api/controllers/authentication.py

```diff
@@ -0,0 +1,31 @@
+import time
+import logging
+from jose import JWTError, jwt
+from flask import current_app
+from werkzeug.exceptions import Unauthorized
+from pds_doi_service.core.util.config_parser import DOIConfigUtil
+
+config = DOIConfigUtil().get_config()
+
+JWT_ISSUER = config.get('API_AUTHENTICATION', 'jwt_issuer')
+JWT_SECRET = config.get('API_AUTHENTICATION', 'jwt_secret')
+JWT_LIFETIME_SECONDS = 600
```

Comment:
@tloubrieu-jpl , is the JWT_LIFETIME_SECONDS variable used? If this is a life time of a JWT access token, then the default value should be 3600 seconds.

Reply:
I updated it